scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,22 @@
+pnpm -F backend-integration test -- {dev,user_profile}.spec
+
+pnpm -F backend-integration test -- proposal_review.spec -t "create proposal review"
+
+pnpm -F backend-integration test -- proposal_review.spec -t "update proposal review batch one"
+pnpm -F backend-integration test -- proposal_review.spec -t "update proposal review batch two"
+
+pnpm -F backend-integration test -- proposal_review.spec -t "list proposal reviews batch one"
+pnpm -F backend-integration test -- proposal_review.spec -t "list proposal reviews batch two"
+pnpm -F backend-integration test -- proposal_review.spec -t "list proposal reviews batch three"
+
+pnpm -F backend-integration test -- proposal_review.spec -t "get proposal review"
+
+pnpm -F backend-integration test -- proposal_review_commit.spec -t "create proposal review commit batch one"
+pnpm -F backend-integration test -- proposal_review_commit.spec -t "create proposal review commit batch two"
+
+pnpm -F backend-integration test -- proposal_review_commit.spec -t "update proposal review commit batch one"
+pnpm -F backend-integration test -- proposal_review_commit.spec -t "update proposal review commit batch two"
+# pnpm -F backend-integration test -- proposal_review_commit.spec -t "update proposal review commit batch three"
```

Comment:
At this point, it completely craps out again, despite the batching so we'll need to run these tests manually for now. I've run them all locally and they pass fine for me.